### DISCUSSION TITLE
Golden Soroban Escapade - Submission

### DISCUSSION THREAD

**Discussion by TimmyC317** [stellar/sorobanathon#152] [General]
Below are my NFT submissions for the Sorobanathon:

PROLOGUE - https://nft.fimi.market/detail/nft/Stellar/GBAXFF6PFYN4DUSLMKUMFOLOLCYMSTFO7JBUSRLUUTQAN574XFSCJVV3/GLDN

CHAPTER 1 - https://nft.fimi.market/detail/nft/Stellar/GCLP5GLFIPBBWD4MQVW46VIP2LAASHEZRCV2CMFAK4REGD36XF2YGB5L/HLWD

CHAPTER 2 - https://nft.fimi.market/detail/nft/Stellar/GADSJUQ7Q3RS2NW4BL62WAHFM3U5YQ363PKDRX6MGYV6N6BDESK523LE/SNEK

CHAPTER 3 - https://nft.fimi.market/detail/nft/Stellar/GBXYAAPNTOQ5OYDGISKEOTNHUWUQPA3A6OOXIX27PQ62MTJXR23ALTZM/NEXT

CHAPTER 4 - https://nft.fimi.market/detail/nft/Stellar/GC3PD3EJRVEUF5JACGDYEUQ32DHQX6EPYDXMC2Y5WQHZURQQ4X3W26FY/SRBN

CHAPTER 5 - https://nft.fimi.market/detail/nft/Stellar/GAQUKIX633EHCUHBQE3DA3RVNNJPWNIJJYY7CTSE5T4CALHKHFX6LAAA/ISLA

CHAPTER 6 - https://nft.fimi.market/detail/nft/Stellar/GB45V4K4MAUU5IBEJJQJDOOTLIKLIEYIQP2X5F2DQSX2MMIDMCONMPJI/FISH

Hope you like!

**Comment by stellarjohn** [comment]
Hi [TimmyC317](https://github.com/TimmyC317)!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
